src/test/unit/controls/Line/Line-spec.js

```diff
@@ -2215,6 +2216,101 @@ describe("Line", () => {
                 });
             });
         });
+        describe("Check if region same for multiline with same dataset", () => {
```

Comment:
Dataset is a term used for data values, in your case the dataset is not same is it? `inputPrimary` and `inputSecondary` have different values. Might want to update the description here.

Reply:
ok i will update the description.